packages/meteor-gazelle-createdby/lib/model.js

```diff
@@ -0,0 +1,32 @@
+Astro.createBehavior({
+  name: 'createdBy',
+  events: {
+    addbehavior: function (behaviorData) {
+      var self = this;
+
+      self.addFields({
+        createdBy: {
+          type: 'string',
+          default: null
+        },
+        updatedBy: {
+          type: 'string',
+          default: []
+        }
+      });
+
+      self.addEvents({
+        beforeInsert: function () {
+          self.createdBy = Meteor.userId();
+        },
+        beforeUpdate: function () {
+          self.updatedBy.push(Meteor.userId());
```

Comment:
@ajax-wcd May I ask where the revision history code might be found (if it exists)? If we have `updatedAt` timestamps in the revision history, then I guess it would make sense to store the `updatedBy` information there, too, and it wouldn't be necessary here.